### PR TITLE
[FIX] Threading stack overflow size

### PR DESCRIPTION
**Pull request by lasley**
* Fix threading stack overflow size from python-pillow/Pillow#1935

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -132,6 +132,10 @@ RUN if [ "$ODOO_VERSION" != "10.0" ]; \
         ln -s /usr/local/bin/openerp-server /usr/local/bin/odoo; \
     fi
 
+# Fix stack size overflow
+# https://github.com/python-pillow/Pillow/issues/1935
+RUN awk '/import/ { print; print "import threading; threading.stack_size(4*80*1024)"; next }1' /usr/local/bin/odoo
```
Comment (tedsalmon) [inline Dockerfile]: Awk doesn't edit in place; you'll need to redirect stdout to the file in question

**Review by tedsalmon — APPROVED** [review]
Unless awk is different in alpine; there's a change required :o

Other than that, LGTM! 👍

File: Dockerfile (hunk 1 above)
Comment (lasley) [reply Dockerfile]: Ahh yup, I obviously didn't test after I switched from simply copying a new odoo-bin!

**Comment by lasley** [comment]
Awk fixed

**Pull request merged**